src/login/login-config.jsx

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react'
```

Comment:
1. Since we don't use any JSX inside of this file, shouldn't we change its extension to `.js`?
2. Let's extract the `clientId` and `cookiePolicy` constants in a separate `config.js` file and export them inside of an object
3. Shouldn't we treat the failure case the same way as the success case? (meaning a separate `useFailure` hook)

Reply:
Done